__tests__/app.test.js

```diff
@@ -66,5 +65,40 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe("bad request");
       });
   });
-
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: responds with updated article", () => {
+    const incrementBy = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(incrementBy)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 105,
+        });
+      });
+  });
+  test("status 400: responds with bad request when invalid input", () => {
+    return request(app)
+      .patch("/api/articles/cats")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("status 404: not found when passed an invalid endpoint", () => {
```

Comment:
Just missing a few tests

- [ ] 404 article not found e.g. "/api/articles/99999"
- [ ] 400 inc_vote not an integer e.g. {inc_votes: "not_a_number"}
- [ ] 200/400 inc_vote key missing - either respond with an error or make no change. e.g. {} or {other_key: 10}